airbyte-integrations/connectors/source-shopify/source_shopify/auth.py

```diff
@@ -34,7 +40,11 @@ def get_auth_header(self) -> Mapping[str, Any]:
         auth_method: str = credentials.get("auth_method")
 
         if auth_method in ["oauth2.0", "access_token"]:
-            return {auth_header: credentials.get("access_token")}
+            access_token = credentials.get("access_token")
+            if access_token:
+                return {auth_header: credentials.get("access_token")}
```

Comment:
nit: on line 43 we are getting `access_token` we can just use the variable value instead of making another `get()`

Reply:
Oh, actually it should be exactly what you suggested, missed getting it back. Corrected! Thanks.